recipe/meta.yaml

```
@@ -40,58 +40,61 @@ outputs:
       run:
         - python >=3.8
 
-        # dependencies matched to pip
+        # dependencies matched to setup.cfg
+        - app-model>=0.1.0,<0.3.0a
```

From v0.4.17's setup.cfg: https://github.com/napari/napari/blob/878f1233e3c0b40afef062e711d8238fe4b44b84/setup.cfg#L46

```suggestion
        - app-model>=0.8.0,<0.3.0
```

Looks like conda-forge doesn't have anything past 0.1.0, so the alpha character probably doesn't matter?

---

We usually put the alpha because in conda versioning 3.0a is smaller than 3.0 😬

---

```suggestion
        - app-model>=0.0.8,<0.3.0a
```

I guess you mean [this](https://github.com/napari/napari/blob/878f1233e3c0b40afef062e711d8238fe4b44b84/setup.cfg#L46), right?